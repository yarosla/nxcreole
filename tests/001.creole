This is a test
  file.

 Test **file //this// is**.
