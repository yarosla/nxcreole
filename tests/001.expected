<p>This is a test
file.</p>
<p>Test <strong>file <em>this</em> is</strong>.</p>
